src/Polly.Core.Benchmarks/Internals/Helper.CircuitBreaker.cs

```diff
@@ -0,0 +1,38 @@
+#pragma warning disable S4225 // Extension methods should not extend "object"
```

Comment:
Is this an artifact from a previous refactor? I can't see any extension methods declared here.